MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 include README.md
 include LICENSE
-include requirements.txt
+include requirements/requirements.txt
 recursive-exclude * *.py[co]
 include versioneer.py
```

Comment:
This "versioneer.py" should be skipped, shouldn't it?

Reply:
I am not completely sure but I don't think so.
That file is called by `setup.py` so I think we should include it in the package. Besides this line had been added by versioneer's configuration itself, thus it probably has a goal.